hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/workflows/PreCheckException.java

```diff
@@ -53,4 +53,10 @@ public ResponseCodeEnum responseCode() {
     public String toString() {
         return "PreCheckException{" + "responseCode=" + responseCode + '}';
     }
+
+    public static void validateTruePreCheck(boolean condition, ResponseCodeEnum errorStatus) throws PreCheckException {
```

Comment:
This probably belongs in the same utilities as the other similar methods.  Adding it to the exception class isn't ideal.

Reply:
We have one similar for HandleException in the `HandleException` class. Followed same pattern